cloud/keystone.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/cloudwan/gohan/schema"
 	"github.com/gophercloud/gophercloud"
 	"github.com/gophercloud/gophercloud/openstack"
+	"github.com/gophercloud/gophercloud/openstack/identity/v3/domains"
```

Comment:
How about aliasing this to `v3domains` for consistency with 2 imports below this one?

Reply:
why not